src/components/EmptyState/EmptyState.css

```diff
@@ -1,34 +1,339 @@
 .empty_state_wrapper {
-    height: 100vh;
-    width: 100%; 
-    display: flex;
-    align-items: center;
-    z-index: -1;
-    position: absolute;
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  z-index: -1;
+  position: absolute;
 }
 
 .empty_state {
-    display: flex;
-    flex-direction: column;
-    text-align: center;
-    align-items: center;
-    margin: 0 auto;
-  }
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  margin: 0 auto;
+}
 
-  .empty_state h1 {
-    font-family: 'quebecbold', Helvetica, Sans-Serif;
-    font-weight: normal;
-    margin: 10px auto;
-  }
+.empty_state h1 {
+  font-family: 'gilroyextrabold', Helvetica, Sans-Serif;
+  font-weight: normal;
+  margin: 10px auto 20px auto;
+  font-size: 27px;
+  letter-spacing: 1.05px;
+}
 
-   .empty_state h4 {
-    font-family: 'quebecregular', Helvetica, Sans-Serif;
-    font-weight: normal;
-    margin: 5px auto;
-  }
+.empty_state p {
+  font-family: 'rennerbook', Helvetica, Sans-Serif;
+  font-weight: normal;
+  margin: 6px auto;
+  color: #4d4d4d;
+}
 
 .empty_state img {
-    height: 220px;
-    width: 220px;
-    text-align: center;
-  }
\ No newline at end of file
+  height: 220px;
+  width: 220px;
+  text-align: center;
+}
+
+.clearfix {
+  display: inline-block;
+}
+.clearfix::after {
+  content: '';
+  visibility: hidden;
+  display: inline-block;
+  font-size: 0;
+  clear: both;
+  height: 0;
+}
+.tekst {
+  color: #3a2012;
+  font-family: Arial, sans-serif;
+  text-transform: uppercase;
+  font-size: 14px;
+  line-height: 28px;
+  font-weight: bold;
+  display: block;
+  letter-spacing: 2px;
+  max-width: 100%;
+  text-align: center;
+  position: absolute;
+  bottom: 20px;
+  left: 30px;
+  z-index: 99;
+}
+a {
+  color: #d67b32;
+  text-decoration: none;
+  cursor: pointer;
+}
+a:hover {
+  text-decoration: underline;
+}
```

Comment:
I don't think we want a globally scoped rule that ill apply to all links coming in.... what is this for, and where does it come from?

Reply:
I think that's a mistaken added line. we can remove it.